indico/modules/events/templates/management/_event_location.html

```diff
@@ -0,0 +1,24 @@
+{% macro render_event_location_info(data) -%}
+    <div id="event-location">
+        <dl class="i-data-list">
+            <dt>{% trans %}Location{% endtrans %}</dt>
+            <dd>
+                {% if data.venue -%}
+                    {{ data.venue_name }}
+                {%- endif %}
+            </dd>
+            <dt>{% trans %}Room{% endtrans %}</dt>
+            <dd>
+                {% if data.room -%}
+                    {{ data.room_name }}
+                {%- endif %}
+            </dd>
+            <dt>{% trans %}Address{% endtrans %}</dt>
+            <dd>
+                {% if data.address -%}
```

Comment:
Why? `{{ data.address }}` should be enough and if it's None (I don't think so) you could use ``{{ data.address or '' }}``